test/end2end/_conformance.py

```
@@ -0,0 +1,106 @@
+    Subscribes to the FakeBus catch-all so the full ordered sequence — entry,
+    activation, dispatch, terminal events — is captured regardless of which
+    clause a given test inspects.
+    """
+    recs: List[Message] = []
+
+    def _rec(serialized):
+        try:
+            recs.append(Message.deserialize(serialized)
+                        if isinstance(serialized, str) else serialized)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)